src/core/src/Axis.cpp

```diff
@@ -128,6 +152,31 @@ namespace iDynTree
 
         return derivative_nonRotated_T_rotated;
     }
+    
+    TransformDerivative Axis::getTranslationTransformDerivative(const double dist) const
```

Comment:
You are right, please review the changes here https://github.com/robotology/idyntree/pull/269/commits/ffbd7ff229cd5b17840c3ac0a2b72eab9eec7448